packages/wallets/src/features/cashier/modules/CashierLocked/CashierLocked.tsx

```diff
@@ -30,7 +31,7 @@ const CashierLocked: React.FC<React.PropsWithChildren> = ({ children }) => {
 
     const isCashierLocked = status?.is_cashier_locked;
```

Comment:
v. minor / optional: so right now isCashierLocked variable might be true, while the account is virtual, so its not locked. Might be worth moving the isVirtual condition to this line to have the "is cashier locker" logic in one place, 

const isCashierLocked = status?.is_cashier_locked && !isVirtual;

Reply:
Updated